src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/TensorsData.cs

```diff
@@ -63,11 +64,21 @@ internal static TensorsData CreateFromNativeHandle(IntPtr handle)
         public int Count
         {
             get {
+                NNStreamer.CheckNNStreamerSupport();
+
                 if (_count != Tensor.InvalidCount)
                     return _count;
 
                 NNStreamerError ret = NNStreamerError.None;
-                ret = Interop.Util.GetTensorsCount(_handle, out int count);
+                int count = 0;
+                try
+                {
+                    ret = Interop.Util.GetTensorsCount(_handle, out count);
+                }
+                catch (DllNotFoundException e)
```

Comment:
DllNotFound is already checked in CheckNNStreamerSupport()

Reply:
ACK